autoscheduler/frontend/src/components/SchedulingPage/ConfigureCard/ConfigureCard.tsx

```diff
@@ -108,29 +108,6 @@ const ConfigureCard: React.FC = () => {
         dispatch(selectSchedule(0));
         if (isMounted.current) setLoading(false);
       });
-
-    // make request to save course cards
-    const courseData: SerializedCourseCardOptions[] = [];
```

Comment:
Yeah, I think just having the 15-second autosave seems like a better solution since when you click generate schedules, it'll automatically save your courses in the next 5-15 seconds anyways